### PR TITLE
Move containers documentation to regular documentation

### PR DESCRIPTION
**Pull request by Martchus**
* Change format from Markdown to AsciiDoc
* Use proper cross-references
* Fix minor details/typos
* Mention that the `docker-compose` setup is still experimental
* See https://progress.opensuse.org/issues/76990

### CODE REVIEW

**Review by okurz — APPROVED** [review]

**Comment by okurz** [comment]
@ilausuch

File: docs/ContainerizedSetup.asciidoc [hunk 1]
```diff
@@ -131,39 +137,34 @@ ports:
...
+   - "443:443"
+   ```
+   You can change the exported port if 443 is already used in your computer, for
+   instance:
+   ```
+   - "10443:443"
+   ```
+
+2. Provide an SSL certificate:
+   ```
```
Comment (ilausuch) [inline docs/ContainerizedSetup.asciidoc:156]: This volumes has to be in the section volumes of the docker-compose service but this section is currently missing on the yaml file and cannot be included if there are not volumes. So please include "volumes:" before these two lines.

**Review by ilausuch — CHANGES_REQUESTED** [review]

File: docs/ContainerizedSetup.asciidoc (hunk 1 above)
Comment (Martchus) [reply docs/ContainerizedSetup.asciidoc:156]: The YAML looks like this:

```
    ports:
      - "9526:9526"
      # Uncomment for SSL access
      # - "443:443"
    environment:
      replicas: ${OPENQA_WEBUI_REPLICAS}
      # If is needed to set a certificate for SSL
      # - cert.crt:/etc/ssl/certs/opeqa.crt
      # - cert.key:/etc/ssl/certs/opeqa.key
```

So I would assume the documentation's first point asks me to uncomment the first comment line and the second point asks me to uncomment the two last comments. Not sure how volumes would come into play here. How would the YAML look like after following step 2? Maybe we should change the YAML so one has to uncomment lines if that's not already possible.

**Comment by okurz** [comment]
https://github.com/os-autoinst/openQA/pull/3822 for the UI test instability. You can retrigger for this PR or rebase on my changes.

File: docs/ContainerizedSetup.asciidoc (hunk 1 above)
Comment (ilausuch) [reply docs/ContainerizedSetup.asciidoc:156]: I add this change here https://github.com/os-autoinst/openQA/pull/3821#issuecomment-811774254 but I am not sure is the place, feel free to add this here

Comment (Martchus) [reply docs/ContainerizedSetup.asciidoc:156]: I would keep my PR as-is and when you do changes which require documentation updates you should to these documentation updates as part of these changes.

Comment (ilausuch) [reply docs/ContainerizedSetup.asciidoc:156]: ok

**Review by ilausuch — APPROVED** [review]

**Pull request merged**